src/main.js

```diff
@@ -25,4 +25,5 @@ function App() {
   );
 }
 
+YellowBox.ignoreWarnings(['Setting a timer']);
```

Comment:
Temporary?

Reply:
Nah, I was sneaking it in here hehe. We set a timer for longer than `react-native` likes. Could make a wrapper which would start another timer after the last one had finished. But that's just as bad as this line here. I don't think it'll be fixed soon and saves me having to put `console.disableYellowBox = true` every time I make a new branch 🤣

Reply:
oh I'm also following the `react-native` repo about the issue.. never going to be changed